<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="1" ignored="0" total="8" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="112" started-at="2022-10-19T06:51:32Z" finished-at="2022-10-19T06:51:32Z">
    <groups>
      <group name="A1">
        <method signature="DependsOnGroupsTestNg_7.login()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="login" class="TestNgRevision.DependsOnGroupsTestNg_7"/>
      </group> <!-- A1 -->
      <group name="test">
        <method signature="DependsOnGroupsTestNg_7.B()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="B" class="TestNgRevision.DependsOnGroupsTestNg_7"/>
        <method signature="DependsOnGroupsTestNg_7.C()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="C" class="TestNgRevision.DependsOnGroupsTestNg_7"/>
      </group> <!-- test -->
      <group name="test1">
        <method signature="DependsOnGroupsTestNg_7.D()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="D" class="TestNgRevision.DependsOnGroupsTestNg_7"/>
        <method signature="DependsOnGroupsTestNg_7.A()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="A" class="TestNgRevision.DependsOnGroupsTestNg_7"/>
      </group> <!-- test1 -->
      <group name="C1">
        <method signature="DependsOnGroupsTestNg_7.contactsTest()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="contactsTest" class="TestNgRevision.DependsOnGroupsTestNg_7"/>
      </group> <!-- C1 -->
      <group name="B1">
        <method signature="DependsOnGroupsTestNg_7.homeTest()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="homeTest" class="TestNgRevision.DependsOnGroupsTestNg_7"/>
      </group> <!-- B1 -->
    </groups>
    <test name="Default test" duration-ms="112" started-at="2022-10-19T06:51:32Z" finished-at="2022-10-19T06:51:32Z">
      <class name="TestNgRevision.DependsOnGroupsTestNg_7">
        <test-method status="PASS" signature="setup()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="setup" is-config="true" duration-ms="13" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="D()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="D" duration-ms="8" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- D -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="tearDown" is-config="true" duration-ms="1" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="setup" is-config="true" duration-ms="0" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="login()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="login" duration-ms="10" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:305)
	at org.testng.Assert.assertEquals(Assert.java:315)
	at TestNgRevision.DependsOnGroupsTestNg_7.login(DependsOnGroupsTestNg_7.java:17)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="tearDown" is-config="true" duration-ms="1" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="setup" is-config="true" duration-ms="0" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="logout()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="logout" duration-ms="3" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="tearDown" is-config="true" duration-ms="1" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="setup" is-config="true" duration-ms="1" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="A()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="A" duration-ms="4" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- A -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="tearDown" is-config="true" duration-ms="1" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="B()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="B" duration-ms="0" depends-on-groups="A1" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method DependsOnGroupsTestNg_7.B()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034] depends on not successfully finished methods in group "A1"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method DependsOnGroupsTestNg_7.B()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034] depends on not successfully finished methods in group "A1"
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1068)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- B -->
        <test-method status="SKIP" signature="C()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="C" duration-ms="0" depends-on-groups="A1" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method DependsOnGroupsTestNg_7.C()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034] depends on not successfully finished methods in group "A1"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method DependsOnGroupsTestNg_7.C()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034] depends on not successfully finished methods in group "A1"
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1068)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- C -->
        <test-method status="SKIP" signature="homeTest()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="homeTest" duration-ms="0" depends-on-groups="A1" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method DependsOnGroupsTestNg_7.homeTest()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034] depends on not successfully finished methods in group "A1"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method DependsOnGroupsTestNg_7.homeTest()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034] depends on not successfully finished methods in group "A1"
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1068)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homeTest -->
        <test-method status="SKIP" signature="contactsTest()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034]" name="contactsTest" duration-ms="0" depends-on-groups="B1" started-at="2022-10-19T12:21:32Z" finished-at="2022-10-19T12:21:32Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method DependsOnGroupsTestNg_7.contactsTest()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034] depends on not successfully finished methods in group "B1"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method DependsOnGroupsTestNg_7.contactsTest()[pri:0, instance:TestNgRevision.DependsOnGroupsTestNg_7@ed9d034] depends on not successfully finished methods in group "B1"
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1068)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- contactsTest -->
      </class> <!-- TestNgRevision.DependsOnGroupsTestNg_7 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
